pkgs/gamescope/default.nix

```diff
@@ -0,0 +1,37 @@
+{ gamescope'
+, fetchpatch
+, fetchFromGitHub
+, glm
+, gbenchmark
+}:
+
+# NOTE: vendoring gamescope for the time being since we want to match the
+#       version shipped by the vendor, which does not work without the kernel
+#       version coherent with the version as shipped by the vendor.
+
```

Comment:
This comment is now partially untrue... We may want to rephrase it so it's only about keeping the version coherent with the shipped vendor version. The issues with it working only with the vendor kernel has been resolved since.

(I'll handle it later today.)